src/MCPServer/lib/assets/workflow.json

```diff
@@ -6355,6 +6355,38 @@
                 "sv": "Ladda upp DIP"
             }
         },
+        "d51a7ed8-9cf4-424b-9671-85fd8b5b95aa": {
+            "config": {
+                "@manager": "linkTaskManagerDirectories",
+                "@model": "StandardTaskConfig",
+                "arguments": "\"%SIPUUID%\" \"%clientAssetsDirectory%premis/premis.xsd\" \"%SIPDirectory%metadata/premis.xml\"",
+                "execute": "loadPREMISEventsFromXML_v1.0",
+                "filter_file_end": null,
+                "filter_file_start": null,
+                "filter_subdir": null,
+                "stderr_file": null,
+                "stdout_file": null
+            },
+            "description": {
+                "en": "Load PREMIS events from metadata/premis.xml"
+            },
+            "exit_codes": {
+                "0": {
+                    "job_status": "Completed successfully",
+                    "link_id": "a536828c-be65-4088-80bd-eb511a0a063d"
+                }
+            },
+            "fallback_job_status": "Failed",
+            "fallback_link_id": "a536828c-be65-4088-80bd-eb511a0a063d",
```

Comment:
It sounds from discussion like the failure case should fail the transfer — if so we should probably link to `61c316a6-0a50-4f65-8767-1f44b1eeb6dd` here.

Reply:
I just asked Evelyn and Peter about this and they think it's OK to fail only the job and keep the transfer workflow going. I'm also going to change the script to error under some other conditions like duplicated event UUIDs being used.